MaxText/train.py

```diff
@@ -345,6 +391,10 @@ def train_loop(config, state=None):
 
   example_batch = None
   last_step_completion = datetime.datetime.now()
+  if config.enable_mllog:
```

Comment:
probably wrap this in a maxtext specific version to preserve readability in train.py.

Reply:
Good suggestion! Will do.

Reply:
Disabled all mllogs in main branch and will cover it in a specific version/branch.